pkg/controllers/managementuser/cis/clusterScanHandler.go

```diff
@@ -461,3 +487,39 @@ func (csh *cisScanHandler) getCISBenchmarkCatalogID() (string, error) {
 func (csh *cisScanHandler) getRancherCISBenchmarkTemplateID() string {
 	return fmt.Sprintf("%s-%s", cutils.SystemLibraryName, templateName)
 }
+
+func (csh *cisScanHandler) isCISv2Enabled(cluster *v3.Cluster) (bool, error) {
+	cfg, err := csh.getKubeConfig(cluster)
```

Comment:
On the UserContext there is a `APIExtClient` which should be used to determine what CRDs are in the downstream cluster.

Reply:
Do I need to initialize APIExtClient? I dont see APIExtClient exposing methods...

Reply:
`user.APIExtClient.ApiextensionsV1().CustomResourceDefinitions().Get()`